index.js

```diff
@@ -3,6 +3,8 @@
 const assert = require('assert');
 const util = require('util');
 
+const deprecate = require('depd')('bem-cell');
```

Comment:
- you cant disable console.log throw ENV
- its colorized
- deprecated is always need :)